dagster.Dockerfile

```
@@ -27,6 +27,10 @@ LABEL org.opencontainers.image.licenses="(EUPL-1.2)"
 
 EXPOSE 3000
 
+COPY scripts/ /opt/dagster/app/scripts/
```

(See the comment about background jobs below.)

```suggestion
ARG TARGETARCH
ENV TINI_VERSION=v0.19.0
ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${TARGETARCH} /tini
RUN chmod +x /tini && /tini --version

COPY scripts/ /opt/dagster/app/scripts/

ENTRYPOINT [ "/tini", "--", "/opt/dagster/app/scripts/start_runs_termination_script_in_background.sh" ]
```